docs/docs/behaviors/tap-dance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Tap-Dance Behavior
+sidebar_label: Tap-Dance
+---
+
+## Summary
+
+A tap-dance key invokes a different behavior (e.g. `kp`) corresponding to how many times it is pressed.
+Tap-dances are completely custom, so for every unique tap-dance key,
+a new tap-dance must be defined in your keymap's `behaviors`.
```

Comment:
Nice work! 

I think a brief example could be useful here, before getting into the technical stuff:

```suggestion
A tap-dance key invokes a different behavior (e.g. `kp`) corresponding to how many times it is pressed. For example, you could configure a tap-dance key that toggles [Caps Word](../caps-word) if tapped once, or Caps _Lock_ if tapped twice.

Tap-dances are completely custom, so for every unique tap-dance key,
a new tap-dance must be defined in your keymap's `behaviors`.
```

I'm not sure if that's the best example, but I tried to think of something practical for those who might not understand the potential use cases of tap-dance.

Reply:
> For example, you could configure a tap-dance key that toggles
> [Caps Word](../caps-word) if tapped once, or Caps _Lock_ if tapped twice.
> The expandability of the number of [`bindings`](#bindings) attached to a
> particular tap-dance is a great way to add more functionality to a single key,
> especially for keyboards with a limited number of keys.

Expanding upon your work here with an extra qualitative statement before the technical stuff